lib/spack/docs/basic_usage.rst

```diff
@@ -952,44 +783,17 @@ Integration with module systems
    interface and/or generated module names may change in future
    versions.
 
-Spack provides some integration with
-`Environment Modules <http://modules.sourceforge.net/>`__
-and `Dotkit <https://computing.llnl.gov/?set=jobs&page=dotkit>`_ to make
-it easier to use the packages it installed.
-
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Installing Environment Modules
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-In order to use Spack's generated environment modules, you must have
-installed the *Environment Modules* package.  On many Linux
-distributions, this can be installed from the vendor's repository:
-
-.. code-block:: sh
-
-   $ yum     install environment-modules  # (Fedora/RHEL/CentOS)
-   $ apt-get install environment-modules  # (Ubuntu/Debian)
-
-If your Linux distribution does not have
-Environment Modules, you can get it with Spack:
-
-.. code-block:: console
-
-   $ spack install environment-modules
-
-In this case to activate it automatically you need to add the following two
-lines to your ``.bashrc`` profile (or similar):
+Spack provides some integration with `Environment Modules
+<http://modules.sourceforge.net/>`_ to make it easier to use the
+packages it installs.  If your system does not already have
+Environment Modules, see InstallEnvironmentModules_.
```

Comment:
Ah, I see where you moved the section now. You should use this instead:
```
Environment Modules, see :ref:`InstallEnvironmentModules`.
```
Sphinx should automatically replace it with a hyperlink with the text from the section title.